src/InterestPayout.Common/Domain/AssetInterest.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace InterestPayout.Common.Domain
+{
+    public class AssetInterest
+    {
+        public long Id { get; }
+        
+        public string AssetId { get; }
+        
+        public decimal InterestRate { get; }
+        
+        public DateTimeOffset ValidUntil { get; }
+         
+        public DateTimeOffset CreatedAt { get; }
+        
+        public int Version { get; init; }
```

Comment:
As discussed on Telegram let's get rid of versioning since this makes everything too complicated

Reply:
`+`